apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.SingleChannel.cs

```diff
@@ -310,7 +312,8 @@ private void StartStreamingPull()
         // Backoff delay (if present) has already been done; no need to delay here.
         private void HandleStartStreamingPullWithoutBackoff()
         {
-            _pull = _client.StreamingPull(CallSettings.FromCancellationToken(_softStopCts.Token));
+            var callSettings = CallSettings.FromHeader(XGoogRequestParamsHeaderKey, string.Format("subscription={0}", _subscriptionName));
```

Comment:
Please add a TODO to call CallSettings.FromGoogleRequestParamsHeader when https://github.com/googleapis/gax-dotnet/issues/733 is fixed and released, and change the second argument to:

```csharp
$"subscription={Uri.EscapeDataString(_subscriptionName.ToString())}"`
```

We could potentially create this whole CallSettings once as a field, but I guess we're not calling HandleStartStreamingPullWithoutBackoff that often...